bucket/mob.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.1.0",
+    "description": "Swift git handover with mob",
+    "homepage": "https://github.com/remotemobprogramming/mob",
```

Comment:
Mob seems to have a homepage here

https://mob.sh/

You might have to update checkver here.
Refer to https://github.com/ScoopInstaller/Main/blob/master/bucket/isx.json if required

Reply:
Thanks for the hint. 
I updated the homepage and checkver